### PR TITLE
Proposed make file changes

### PR DESCRIPTION
**Pull request by haerdib**
instead of `make benchmark` we can run it with `BENCHMARK=1 make`... what do you think @zktony  ?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -197,70 +198,7 @@ $(Signed_RustEnclave_Name): $(RustEnclave_Name)
 	@echo "SIGN =>  $@"
 	@echo
 	@echo "Enclave is in $(SGX_ENCLAVE_MODE)"
-.PHONY: benchmark
-benchmark: $(Client_Name) $(Worker_Name) $(Signed_RustEnclave_Name)
```
Comment (simonsso) [inline Makefile:201]: Keep target benchmark and print an error message about new syntax

File: Makefile [hunk 2]
```diff
@@ -197,70 +198,7 @@ $(Signed_RustEnclave_Name): $(RustEnclave_Name)
...
-
-.PHONY: enclave-benchmark
-enclave:
-	@echo
-	@echo "Building the enclave"
-	$(MAKE) -C ./enclave/ benchmark
-
-.git/hooks/pre-commit: .githooks/pre-commit
-	@echo "Installing git hooks"
-	cp .githooks/pre-commit .git/hooks
```
Comment (simonsso) [inline Makefile:263]: why is this removed - git hooks sounds unrelated

**Review by simonsso — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (haerdib) [reply Makefile:263]: Because I removed the whole enclave-benchmark command. (see the enclave command right below (L265-L272))

File: Makefile (hunk 1 above)
Comment (haerdib) [reply Makefile:201]: Hm, is this really necessary? We have not yet merged anything to the develop branch, so `make benchmark` itself would be a new syntax.

Comment (haerdib) [reply Makefile:201]: Ah, please note, this is a PR to the branch of @zktony , not to develop.

**Comment by zktony** [comment]
LGTM, @simonsso @haerdib can we merge it now?

**Comment by haerdib** [comment]
> 
> 
> LGTM, @simonsso @haerdib can we merge it now?

As it's only to your branch and not to develop, I think its fine.

**Comment by zktony** [comment]
Okay cool .. thnaks for help @haerdib 🙂
